contracts/src/internal/HyperdriveLP.sol

```diff
@@ -383,12 +373,8 @@ abstract contract HyperdriveLP is
             msg.sender, // provider
             _options.destination, // destination
             _lpShares,
-            _convertToBaseFromOption(proceeds, vaultSharePrice, _options), // base proceeds
-            _convertToVaultSharesFromOption(
```

Comment:
Can you get rid of these helper methods now?

Reply:
Yep, I did that